src/pages/Accounts.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="content">
+    <card>
+      <template slot="header">
+        <h5 class="title">Accounts</h5>
+        <p class="category">A component to interact with your accounts</p>
+      </template>
+      <div class="row">
+        <div class="col-lg-12 col-md-3 col-sm-4 col-xs-6 col-xs-6 text-left">
+          <card>
+            <p v-for="(account, address) in accounts">{{ account }}. Account: {{ address }}</p>
+            <hr />
+            <p>
+              Choose an address to convert to:
+              <select
+                v-model="selectedChain"
+                @change="onSelect($event)"
+              >
+                <option v-for="chainname in chains" v-bind:value="chainname">{{ chainname }}</option>
+              </select>
+            </p>
+            <hr />
+            <p
+              v-if="selectDropdown()"
+              v-for="(account, address) in accounts"
+            >{{ account }}. Address on {{ onlyChain }}: {{ address | prefix(onlyChain) }}</p>
+          </card>
+        </div>
+      </div>
+    </card>
+  </div>
+</template>
+<script>
+import { Card } from "@/components/index";
+import { web3Accounts } from "@polkadot/extension-dapp";
+import { MyFunctions } from "../components/Filters/Filter.js";
+import api from "../connection.js";
+const util = require("@polkadot/util-crypto");
+let dappex = require("@polkadot/extension-dapp");
+
+export default {
+  el: "#selector",
+  components: {
+    Card,
+    web3Accounts
+  },
+  data() {
+    return {
+      selected: false,
+      accounts: {},
+      address: {},
+      onlyAddress: "",
+      selectedAddress: "",
+      selectedChain: "",
+      onlyChain: "",
+      chainName: "",
+      chains: [
+        "polkadot (0)",
+        "kusama (2)",
+        "plasm (5)",
+        "bifrost (6)",
+        "edgeware (7)",
+        "karura (8)",
+        "reynolds (9)",
+        "acala (10)",
+        "laminar (11)",
+        "kulupu (16)",
+        "darwinia (18)",
+        "stafi (20)",
+        "robonomics (32)",
+        "centrifuge (36)",
+        "substrate (42)"
+      ]
+    };
+  },
+  methods: {
+    /* selectDropdown: function()
+       This ensures that data is only displayed if the 
+       dropdown has been selected, else return no data.
+    */
+    selectDropdown: function() {
+      if (this.selected == true) return true;
```

Comment:
Is there a way this could be expressed in cleaner code? 🤔

Just `return this.selected` does the same thing 🙂